sklearn/model_selection/tests/test_validation.py

```diff
@@ -134,38 +135,38 @@ def test_cross_val_score():
     for a in range(-10, 10):
         clf.a = a
         # Smoke test
-        scores = cross_val_score(clf, X, y)
-        assert_array_equal(scores, clf.score(X, y))
+        scores = cross_val_score(clf, X, y2)
+        assert_array_equal(scores, clf.score(X, y2))
 
         # test with multioutput y
-        scores = cross_val_score(clf, X_sparse, X)
-        assert_array_equal(scores, clf.score(X_sparse, X))
+        multioutput_y = np.column_stack([y2, y2[::-1]])
```

Comment:
you need not change this.

For y with n_dim > 1, the default strategy is KFold right? So this should not raise a warning